zdict/tests/dictionaries/test_spanish.py

```diff
@@ -0,0 +1,42 @@
+from pytest import raises
+from unittest.mock import Mock, patch
+
+from zdict.dictionaries.spanish import SpanishDict
+from zdict.exceptions import NotFoundError
+
+
+class TestSpansishDict:
+    d = SpanishDict()
+    word = 'Spanish'
+    timeout = 5
+    raw_html = d._get_raw(word, timeout)
+    record = d.query(word, timeout)
```

Comment:
It seems that there is no function requires it but `test_show`.
How about just setup it as local variable of `test_show`?